api/destination.go

```diff
@@ -6,21 +6,21 @@ import (
 )
 
 type Destination struct {
-	ID         uid.ID                `json:"id"`
-	UniqueID   string                `json:"uniqueID" form:"uniqueID" example:"94c2c570a20311180ec325fd56"`
-	Name       string                `json:"name" form:"name"`
-	Kind       string                `json:"kind"`
+	ID         uid.ID                `json:"id" note:"ID of the destination" example:"7a0b06b30F"`
```

Comment:
this example isn't a valid ID, there are no `0` in the id set. has to be 11 characters from the set of `123456789abcdefghijkmnopqrstuvwxyzABCDEFGHJKLMNPQRSTUVWXYZ`

Reply:
does that mean the format for the ID is wrong? that shows up as "[\\da-zA-HJ-NP-Z]{1,11}". i think \d means 0-9 not 1-9

Reply:
but updated the string to use other digits

Reply:
and pr #3671 fixes it